### DISCUSSION TITLE
Snapshot v0 - IS_WEEKLY true on Tuesday?

### DISCUSSION THREAD

**Discussion by mjahammel** [ScalefreeCOM/datavault4dbt#49] [Q&A] [answered]
According to [the documentation ](https://github.com/ScalefreeCOM/datavault4dbt/wiki/Snapshot-Control-v0), the column IS_WEEKLY for a v0 Snapshot should be true only for rows where the sdts column is a Monday.  I've tried several different combinations of start_date and daily_snapshot_time (including the example shown in Example 1), but in all cases IS_WEEKLY is true if the stds date is a Tuesday.  Am I doing something wrong or just misunderstanding the documentation?

Thanks,
Maury

**Comment by tkirschke** [comment]
Hi @mjahammel and thanks for reaching out!

You are right, is_weekly should be set to TRUE for each Monday. I already found the problem in the code and will fix it ASAP:

`        CASE
            WHEN EXTRACT(DAYOFWEEK FROM  sdts) = 2 THEN TRUE
            ELSE FALSE
        END AS is_weekly,`
Here, it should be 1 instead of 2. 

Will create an issue out of this discussion 👍 

Kind regards, 
Tim

**Comment by tkirschke** [comment, marked as answer]
Hi @mjahammel, 

I have fixed this problem and included it in the[ latest release](https://github.com/ScalefreeCOM/datavault4dbt/releases/tag/v1.0.12).

This should fix the problem! Let me know if it works for you, for me it did 😸 

Kind regards, 
Tim
